spk/sonarr/Makefile

```
@@ -1,22 +1,19 @@
-SPK_NAME = nzbdrone
+SPK_NAME = sonarr
 SPK_VERS = $(shell date +%Y%m%d)
-SPK_REV = 20
+SPK_REV = 22
```

as this is a new package, we can start at revision 1
```suggestion
SPK_REV = 1
```